examples/README.md

```diff
@@ -12,12 +12,12 @@ cd examples/
 
 ### Create letters from CSV
 
-An example showing how to validate and clean addresses from a CSV spreadsheet full of shipping addresses using Lob's [Address Verification API](https://lob.com/verification/address) and then using the cleaned, valid addresses to dynamically create sample billing letters with variable data using Lob's [Letter API](https://lob.com/services/letters).
+An example showing how to dynamically create sample billing letters with variable data using Lob's [Letter API](https://lob.com/services/letters).
 
-In order to run the program enter:
+In order to run the program, add your API key to letter.py and enter:
```

Comment:
Thoughts on changing the instructions here? I feel it would be better to have this instruction in between the `cd` and `python`.